pkg/service/transaction/transactionServiceImpl.go

```diff
@@ -105,5 +114,66 @@ func (s *transactionService) Delete(ctx context.Context, id string) (*entities.T
 		return nil, err
 	}
 
+	s.SetTranactionSubAttributes(transaction, sp)
 	return transaction, nil
 }
+
+func (s *transactionService) SetTranactionSubAttributes(tranasctionData any, sp trace.Span) {
```

Comment:
Why you do not use {}interface type for transactionData

Reply:
if use interface type in param , i think do 2 func for tracerSubAtt between []interface and interface

// i don't want have many func HaHaHa😂